### DISCUSSION TITLE
One doubt regarding hosts file

### DISCUSSION THREAD

**Discussion by ramon501** [evilsocket/opensnitch#762] [Q&A]
When i try to run this command on my home folder to set up the hosts file it tells me it needs authentication, is this normal ?
the command:
`mkdir /media/ads-list/ `

**Comment by gustavo-iniguez-goya** [comment]
Hi @ramon501 ,

Yes, because /media/ usually is owned by root. You can either create it in your home directory: `$ mkdir ~/ads-list/` or use sudo to create it under /media/:
```bash
~$ sudo mkdir /media/ads-list/
~$ chown USERNAME:USERNAME /media/ads-list/
```

I'll update the documentation to correct this detail.
